guides/source/asset_pipeline.md

```diff
@@ -734,13 +641,14 @@ as:
 <%= asset_path('smile.png') %>
 ```
 
-Will be rendered as full CDN URLs like `http://mycdnsubdomain.fictional-cdn.com/assets/smile.png`
-(digest omitted for readability).
+Will be rendered as full CDN URLs like
+`http://mycdnsubdomain.fictional-cdn.com/assets/smile.png` (digest omitted for
+readability).
 
 If the CDN has a copy of `smile.png`, it will serve it to the browser, and your
 server doesn't even know it was requested. If the CDN does not have a copy, it
```

Comment:
```suggestion
If the CDN has a copy of `smile.png`, it will serve it to the browser, and the origin
server won't even know it was requested. If the CDN does not have a copy, it
```